.fixtures.yml

```diff
@@ -5,6 +5,6 @@ fixtures:
       ref: '1.0.0'
     stdlib:
       repo: "git://github.com/puppetlabs/puppetlabs-stdlib.git"
-      ref: '2.4.0'
+      ref: '2.5.1'
```

Comment:
yes, but you're not bumping it in [metadata.json](https://github.com/StackExchange/puppetlabs-haproxy/blob/master/metadata.json#L70)

which i called manifest.json for some reason… sorry about that ~_~